### PR TITLE
RHDEVDOCS-3829 Additional ODC release notes

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 4.10 release notes only
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3829
- Direct link to doc preview: https://deploy-preview-42725--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-bug-fixes
- SME review: @debsmita1 @sahil143 @nemesis09 
- QE review: @sanketpathak 
- Peer review: @rishumehra

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -1302,7 +1302,7 @@ This update corrects the accepted label specification so that the plug-in import
...
 
-* Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])    
+* Previously, when using the OVN-Kubernetes cluster network provider, the presence of any unknown field in an internal configuration map could cause the OVN-Kubernetes pods to fail to start during a cluster upgrade. Now the presence of unknown fields causes a warning, rather than a failure. As a result, the OVN-Kubernetes pods now successfully start during a cluster upgrade. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1988440[BZ#1988440])
```
Comment (rishumehra) [inline release_notes/ocp-4-10-release-notes.adoc:1305]: Rephrase suggestion:

... the presence of any unknown field in an internal configuration map could fail to start the OVN-Kubernetes pods to fail to start during a cluster upgrade.

**Review by rishumehra — APPROVED** [review]
A minor suggestion. Rest seems fine, pre-approving the PR.

File: release_notes/ocp-4-10-release-notes.adoc (hunk 1 above)
Comment (rolfedh) [reply release_notes/ocp-4-10-release-notes.adoc:1305]: Updated. Thanks.

Comment (rolfedh) [reply release_notes/ocp-4-10-release-notes.adoc:1305]: But I agree with your edit. Don't have time to follow up on this.

File: release_notes/ocp-4-10-release-notes.adoc [hunk 2]
```diff
@@ -1770,7 +1770,13 @@ In various circumstances, the management state of the Cluster Samples Operator c
...
+
+* Before this update, the tooltip for tasks in the *PipelineRun details* showed misleading information. It showed the time elapsed since the task ran, not how long they ran. For example, it showed 122 hours for a task that ran for a couple of seconds 5 days ago. With this update, the tooltip shows the duration of the task. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011368[BZ#2011368])
+
+* Before this update, a layout issue caused the Web terminal to open outside the view and resize incorrectly. This update fixes the layout issue by updating the CSS. Now, the terminal opens inside the main view and resizes appropriately. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2048598[BZ#2048598])
+
+* Before this update, a filtering issue displayed irrelevant and invalid tabs for workloads in the *Topology* view. This update fixes the issue by filtering those tabs before rendering the *Topology* view. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2053252[BZ#2053252])
```
Comment (nemesis09) [inline release_notes/ocp-4-10-release-notes.adoc:1779]: I don't think this issue was introduced and fixed in 4.10 itself and did not exist in prior release.
It must have been an oversight on my part.
Please remove this bug fix from the notes.

Comment (nemesis09) [inline release_notes/ocp-4-10-release-notes.adoc:1779]: I don't think this issue was introduced and fixed in 4.10 itself and did not exist in prior release.
It must have been an oversight on my part.
Please remove this bug fix from the notes.

Comment (rolfedh) [reply release_notes/ocp-4-10-release-notes.adoc]: Removed. Thanks.

**Review by nemesis09 — APPROVED** [review]
Thank you for the changes.
/lgtm

**Pull request merged**